src/kic/datamodel/models/core.py

```diff
@@ -183,3 +229,22 @@ class Meta:
         verbose_name = "object-verzoek"
         verbose_name_plural = "object-verzoeken"
         unique_together = ("verzoek", "object")
+
+
+class VerzoekProduct(APIMixin, models.Model):
+    uuid = models.UUIDField(
+        unique=True, default=uuid.uuid4, help_text="Unieke resource identifier (UUID4)"
+    )
+    verzoek = models.ForeignKey(
+        "datamodel.Verzoek",
+        on_delete=models.CASCADE,
+        help_text="URL-referentie naar het VERZOEK.",
+    )
+    product = models.URLField(
+        null=True,
+        blank=True,
+        help_text="URL-referentie naar het PRODUCT (in de Producten en Diensten API).",
+    )
+    product_code = models.CharField(
+        max_length=20, help_text="De unieke code van het PRODUCT."
```

Comment:
missing `blank=True` - also maybe it's worthwile to add a db constraint that either `product` or `product_code` should be filled.

Reply:
Added validation in `clean`.